docs/rfcs/rfc-029-connect-in-update-mutations.md

```diff
@@ -0,0 +1,281 @@
+# Connect on Update
```

Comment:
Perhaps a better name for the RFC (and the file) is something like "Connect if a relationship already exists"?